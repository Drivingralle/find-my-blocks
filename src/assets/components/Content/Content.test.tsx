import React from "react";
import renderer from "react-test-renderer";
import { Content } from ".";

it("renders the Content", () => {
  const tree = renderer.create(<Content>Hello World</Content>).toJSON;
  expect(tree).toMatchSnapshot();
});

it("renders the Content with the different sizes", () => {
  const small = renderer.create(
    <Content spacing="small">Hello World</Content>
  ).toJSON;
  const medium = renderer.create(
    <Content spacing="medium">Hello World</Content>
  ).toJSON;
  const large = renderer.create(
    <Content spacing="large">Hello World</Content>
  ).toJSON;
  expect(small).toMatchSnapshot();
  expect(medium).toMatchSnapshot();
  expect(large).toMatchSnapshot();
});
