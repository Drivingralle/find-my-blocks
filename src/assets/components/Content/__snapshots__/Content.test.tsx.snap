// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Content 1`] = `[Function]`;

exports[`renders the Content with the different sizes 1`] = `[Function]`;

exports[`renders the Content with the different sizes 2`] = `[Function]`;

exports[`renders the Content with the different sizes 3`] = `[Function]`;
